src/services/Auth.service.js

```diff
@@ -0,0 +1,22 @@
+export class AuthService {
+    login(token) {
+        localStorage.setItem('userToken', token);
+
+    }
+
+    getToken() {
+        return localStorage.getItem('userToken');
+    }
+
+    logout() {
+        localStorage.setItem('userToken', null);
+    }
+
+    isLoggedIn() {
```

Comment:
can be simple like this:
`isLoggedIn = () => (this.getToken() === null || this.getToken() === 'null' || !this.getToken())`